docs/_user_guide/slurm.md

```diff
@@ -66,62 +64,60 @@ python3 yahtzee.py 100000
              JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
               1043    normal    a_job    user1 PD       0:00      1 (Resources)
               1042    normal    job_2    user2  R    1:29:53      1 yen11
               1041    normal     bash    user3  R    3:17:08      1 yen11
 ```
 
-Jobs with state (ST) R are running, and PD are pending.  Your job will run based on this queue.
+Jobs with state (ST) R are running, and PD are pending. Your job will run based on this queue.
 
 ## Best Practices
 
 ### Use all of the resources you request
 
-The Slurm scheduler keeps track of the resources you request, and the resources you use. Frequent under-utilization of CPU and Memory will affect your future job priority.  You should be confident that your job will use all of the resources you request.  It's recommended that you run your job on the interactive Yens, and [monitor resource usage](/faqs/howCheckResourceUsage.html) to make an educated guess on resource usage.
+The Slurm scheduler keeps track of the resources you request, and the resources you use. Frequent under-utilization of CPU and Memory will affect your future job priority. You should be confident that your job will use all of the resources you request. It's recommended that you run your job on the interactive Yens, and [monitor resource usage](/_user_guide/best_practices_monitor_usage/){:target="_blank"} to make an educated guess on resource usage.
 
 ### Restructure your job into small tasks
 
-Small jobs start faster than big jobs. Small jobs likely finish faster too.  If your job requires doing the same process many times (i.e. OCR'ing many PDFs), it will benefit you to setup your job as many small jobs.
+Small jobs start faster than big jobs. Small jobs likely finish faster too. If your job requires doing the same process many times (i.e. OCR'ing many PDFs), it will benefit you to setup your job as many small jobs.
 
 ## Tips and Tricks
 
 ### Current Partitions and their limits
 
-Run `sinfo` command to see available partitions:
-
-```bash
-$ sinfo
-```
-
-You should see the following output:
+Run `sinfo` in a terminal to see available partitions:
 
-```bash
+```{.yaml .no-copy title="Terminal Output"}
 USER@yen4:~$ sinfo
 PARTITION AVAIL  TIMELIMIT  NODES  STATE NODELIST
-normal*      up 2-00:00:00      7   idle yen[11-17]
-dev          up    2:00:00      7   idle yen[11-17]
-long         up 7-00:00:00      7   idle yen[11-17]
-gpu          up 1-00:00:00      2   idle yen-gpu[1-2]
+normal*      up 2-00:00:00      5    mix yen[11,14,16-18]
+normal*      up 2-00:00:00      3  alloc yen[12-13,15]
+dev          up    2:00:00      5    mix yen[11,14,16-18]
+dev          up    2:00:00      3  alloc yen[12-13,15]
+long         up 7-00:00:00      5    mix yen[11,14,16-18]
+long         up 7-00:00:00      3  alloc yen[12-13,15]
+gpu          up 1-00:00:00      1    mix yen-gpu1
+gpu          up 1-00:00:00      2   idle yen-gpu[2-3]
 ```
```

Comment:
this output might be confusing because every partition is listed twice (because they have different states so the nodes are listed for each partition-state) ..but we actually only have 4 partitions